tests/x11/addon_products_via_SCC_yast2.pm

```diff
@@ -0,0 +1,66 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: add addon to SLES via SCC
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+use base qw(y2_installbase y2_module_guitest);
+use strict;
+use warnings;
+use testapi;
+use registration;
+use version_utils qw(is_sle is_leap);
+use x11utils 'turn_off_gnome_screensaver';
+use YaST::Module;
+use utils;
+
+sub test_setup {
+    select_console "x11";
+    x11_start_program('xterm');
+    turn_off_gnome_screensaver;
+}
+
+sub run {
+    test_setup;
+    YaST::Module::open(module => 'scc', ui => 'qt');
+    #y2_module_guitest::launch_yast2_module_x11('scc', target_match => [qw(scc-registration packagekit-warning)], maximize_window => 1);
+    save_screenshot;
+    assert_screen("scc-registration", 30);
+    wait_screen_change { send_key 'alt-f10' };
+    save_screenshot;
+    $testapi::distri->get_registration()->register_via_scc({
+            email => get_var('SCC_EMAIL'),
+            reg_code => get_var('SCC_REGCODE')});
+    save_screenshot;
+    my @scc_addons = split ',', get_var('SCC_ADDONS');
+    $testapi::distri->get_module_registration()->register_extension_and_modules([@scc_addons]);
+    save_screenshot;
+
+    # No libyui support on yast_scc-pkgtoinstall page so we have to use assert-screen
+    assert_screen("yast_scc-pkgtoinstall", 100);
+    send_key_until_needlematch 'yast_scc-installation-summary', 'alt-a', 5, 5;
+    $testapi::distri->get_module_registration_installation_report()->press_next();
```

Comment:
no check of the packages with Mojo as suggested in the ticket:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/3d508b145a42f0e46dcf0aef1d11cef223690193/tests/installation/partitioning/suggested_partitioning/verify_decrypted_partition_deleted.pm#L24
we should file a ticket to not forget this later, it might be helpful.

Reply:
> we need to cover all the architectures where this is running, there are different yaml schedules under: `schedule/yast/select_modules_and_patterns+registration/` https://openqa.suse.de/tests/overview?arch=&flavor=Full&machine=&test=&modules=suseconnect_scc&module_re=&distri=sle&version=15-SP4&build=151.1&groupid=129#
> 
> MR for AC2 in https://progress.opensuse.org/issues/108227 ?

For select_modules_and_patterns+registration, run aarch64 and ppc64le result is ok. s390 can't run since 151.1 repo already not exist(Pass case) and we have ticket blocked on latest build.
sle-15-SP5-Full-aarch64-Build15.2-select_modules_and_patterns+registration@aarch64 -> https://openqa.suse.de/t9411438
set priority for 9411438
sle-15-SP5-Full-ppc64le-Build15.2-select_modules_and_patterns+registration@ppc64le -> https://openqa.suse.de/t9411439
set priority for 9411439

https://gitlab.suse.de/qsf-y/qa-sle-functional-y/-/merge_requests/413

Reply:
https://progress.opensuse.org/issues/115973